app/src/main/java/io/intrepid/contest/models/InvitationResponse.java

```diff
@@ -0,0 +1,17 @@
+package io.intrepid.contest.models;
+
+import com.google.gson.annotations.Expose;
+
+import java.util.Date;
+import java.util.UUID;
+
+public class InvitationResponse {
+    @Expose
+    public UUID id;
+    @Expose
+    public UUID contestantId;
```

Comment:
Actually it's not 100% sure we'll have them, because the web team has not defined the response yet. I'll remove these for now, and in my next PR I'll change it anyway based on what I use afterwards.